tests/model_fields/tests.py

```diff
@@ -178,6 +178,22 @@ def get_foo_bar_display(self):
         f = FooBar(foo_bar=1)
         self.assertEqual(f.get_foo_bar_display(), 'something')
 
+    def test_overriding_inherited_FIELD_display(self):
+        class Base(models.Model):
+            foo = models.CharField(max_length=254, choices=[('A', 'Base A'), ('B', 'Base B')])
+
+            class Meta:
+                abstract = True
+
+        class Child(Base):
+            foo = models.CharField(max_length=254, choices=[('A', 'Child A'), ('B', 'Child B'), ('C', 'Child C')])
+
+        tests = ['A', 'B', 'C']
+        for t in tests:
+            with self.subTest(test=t):
+                c = Child(foo=t)
+                self.assertEqual(c.get_foo_display(), 'Child %s' % t)
```

Comment:
I think we can remove `('B', 'Base B')` and `('B', 'Child B')` because it tests the same case as `A`.